src/components/VerticalTabs/VerticalTabs.jsx

```diff
@@ -0,0 +1,138 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, getFirst, findTypes, omitProps } from '../../util/component-types';
+import reducers from './VerticalTabs.reducers';
+import VerticalListMenu from '../VerticalListMenu/VerticalListMenu';
+
+const cx = lucidClassNames.bind('&-VerticalTabs');
+
+const {
+	any,
+	string,
+	number,
+	bool,
+	func,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["navigation"], "madeFrom": ["VerticalListMenu"]}
+ *
+ * `VerticalTabs` provides vertically tabbed navigation. It has a flexible
+ * interface that allows tab content to be passed as regular React children or
+ * through props.
+ */
+const VerticalTabs = createClass({
+	displayName: 'VerticalTabs',
+
+	components: {
+		/**
+		 * Content that will be rendered in a tab. Be sure to nest a Title inside
+		 * each Tab or provide it as a prop.
+		 */
+		Tab: createClass({
+			displayName: 'VerticalTabs.Tab',
+			propName: 'Tab',
+			propTypes: {
+				/**
+				 * Determines if the Tab is selected.
+				 */
+				isSelected: bool,
+			},
+		}),
+		/**
+		 * Titles can be provided as a child or prop to a Tab.
+		 */
+		Title: createClass({
```

Comment:
yeah the nesting of types is incidental to the finding of a particular type in an element's children/props, so nesting is purely a cosmetic thing